lib/mailgun_catcher/version.rb

```diff
@@ -1,3 +1,3 @@
 module MailgunCatcher
-  VERSION = '0.1.0'
+  VERSION = '0.1.1'
```

Comment:
because you are changing the dependency version upgrade this to 0.2
```suggestion
  VERSION = '0.2.0'
```

Reply:
After this, merge it, release it